bin/run-tests.sh

```diff
@@ -24,13 +24,13 @@ if $WATCH_FLAG; then
 	# Change directory to WooCommerce Payments' root in order to have access to .phpunit-watcher.yml
 	docker compose exec -u www-data wordpress bash -c \
 		"cd /var/www/html/wp-content/plugins/woocommerce-payments && \
-		php -d xdebug.remote_autostart=on \
+		php -d xdebug.start_with_request=yes \
 		./vendor/bin/phpunit-watcher watch --configuration ./phpunit.xml.dist $*"
 else
 	echo "Running the tests..."
 
-	docker compose exec -u www-data wordpress \
-		php -d xdebug.remote_autostart=on \
+	docker-compose exec -u www-data wordpress \
+		php -d xdebug.start_with_request=yes \
```

Comment:
If I run this script without listening to Xdebug in my IDE, I see the following warning multiple times:
> [30-Aug-2024 11:47:19 UTC] Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.rancher-desktop.internal:8999 (through xdebug.client_host/xdebug.client_port).

This also happens when running the `bin/check-test-coverage.sh` script, as both of them declare `xdebug.start_with_request=yes`. Did you see the same thing in your local environment?

Reply:
Yes. Fixed it in https://github.com/Automattic/woocommerce-payments/pull/9146/commits/3d7b458a774a53e1c7f25887c9a65f066a21270a and removed direct usages in shell scripts.